build.user.properties.template

```diff
@@ -0,0 +1,21 @@
+nativity.dir=C:/Users/sync/Dev/streem/liferay-nativity
```

Comment:
Yea, but this is now the first place in the codebase that requires a specific username, which is in my opinion way to strict of a restriction.  What happens when we want to build this in different VMs, or on Jenkins machines that don't have the username 'sync'?